core/modules/widgets/vars.js

```diff
@@ -0,0 +1,79 @@
+/*\
+title: $:/core/modules/widgets/vars.js
+type: application/javascript
+module-type: widget
+
+This widget allows the user to set multiple variables in one go.
+It thereby reduces code complexity that would arise when setting
+many variables using "<$set>".
+
+```
+\define helloworld() Hello world!
+<$vars greeting="Hi" me={{!!title}} sentence=<<helloworld>>>
+  <<greeting>>! I am <<me>> and I say: <<sentence>>
+</$vars>
+```
+
+Original discussion that led to the creation of this widget:
```

Comment:
Similarly, I'd say that this reference is redundant. People should look at the git history of the file to see the original commit, and it's there that we link to the discussions.